pkg/operator/ceph/cluster/mgr/mgr.go

```diff
@@ -341,8 +346,8 @@ func (c *Cluster) enablePrometheusModule(clusterName string) error {
 	return nil
 }
 
-// Ceph docs about the dashboard module: http://docs.ceph.com/docs/luminous/mgr/dashboard/
-func (c *Cluster) configureDashboardModule(clusterName string, enable bool) error {
+// Ceph docs about the dashboard module: http://docs.ceph.com/docs/mimic/mgr/dashboard
```

Comment:
Does this setting require mimic? I don't see it in the luminous docs. We are about to move to mimic, but haven't done so quite yet.

Reply:
ok, let's plan on merging this as soon as we update to mimic. I know @rootfs was experimenting with it, so we may have a PR soon.

Reply:
It doesn't require mimic, here is the doc from [Luminous](http://docs.ceph.com/docs/luminous/mgr/dashboard/), I enabled it on my cluster using the toolbox

Reply:
ah thanks for pointing that out, i missed it.

Reply:
Huh, missed that too somehow.